README.md

```diff
@@ -1,3 +1,11 @@
+## Alternatives
+
+This mod is currently stuck on 1.16.x, and as such, will not function on any newer versions, however there are currently 2 alternatives:
+
+1 (recommended option) - [WorldTools](https://github.com/Avanatiker/WorldTools), a Client-sided mod for 1.20.1+ on both forge and fabric, may also work with [ViaFabricPlus](https://github.com/ViaVersion/ViaFabricPlus) to allow downloading worlds from older servers, snapshot and april fools servers, and even bedrock servers
+
+2 - [Minecraft World Downloader](https://github.com/mircokroon/minecraft-world-downloader), an external application that works as a proxy to intercept world data and save it in a minecraft world folder
```

Comment:
WorldTools seems like a more recent project that hasn't been used or downloaded too much. Based on a quick look it sounds reasonable, but I'd rather not promote it as the recommended one between the two. It also seems important to note that these are unrelated projects.

Here's my thoughts on alternative phrasing:

```suggestion
This mod is currently stuck on 1.16.x, and as such, will not function on any newer versions, however as of 2023 there are currently 2 alternatives developed by other people:

* [Avanatiker/WorldTools](https://github.com/Avanatiker/WorldTools), a client-sided mod for 1.20.1+ on both forge and fabric
* [mircokroon/minecraft-world-downloader](https://github.com/mircokroon/minecraft-world-downloader), an external application that works as a proxy to intercept world data and save it in a Minecraft world folder
```

This would render as:

> This mod is currently stuck on 1.16.x, and as such, will not function on any newer versions, however as of 2023 there are currently 2 alternatives developed by other people:
>
> * [Avanatiker/WorldTools](https://github.com/Avanatiker/WorldTools), a client-sided mod for 1.20.1+ on both forge and fabric
> * [mircokroon/minecraft-world-downloader](https://github.com/mircokroon/minecraft-world-downloader), an external application that works as a proxy to intercept world data and save it in a Minecraft world folder

ViaFabricPlus is an interesting thing to mention as well but I don't think it's a good idea to list it as something that "may also work" - I don't want to include something that would get people's hopes up if the WorldTools maintainers don't directly support it (nor do I want to result in confused people bothering them about something that they don't want to support if the developers choose not to support it). I assume there are also some complications regarding that (I assume it would produce a world in the 1.20 format, not the format of the older versions, for instance).

Reply:
Ok, I put the recommended thing there since worldtools is much more similar to WDL than the other one, but that makes sense, thanks!